src/BenchmarksServer/Startup.cs

```diff
@@ -332,6 +332,11 @@ private static Process StartProcess(string hostname, string benchmarksRepo, Serv
                 arguments += $" --connectionFilter {job.ConnectionFilter}";
             }
 
+            if (job.ThreadCount.HasValue)
+            {
+                arguments += $" --threadCount {job.ThreadCount}";
```

Comment:
Suggest `job.ThreadCount.Value` for consistency with `HasValue()` / `Value()` elsewhere in this class.